Fix/App store description overflow CSS

ShobhitAd opened the pull request:
Fixes #440

### Testing Plan
- [x] I have verified that I have not introduced new warnings in this PR (or explain why below)
- [x] I have tested this PR against Core and verified behavior (if applicable, if not applicable, explain why below).

#### Core Tests
Install and uninstall the "Hello Webengine" app

Core version tested against: release/8.0.0 (https://github.com/smartdevicelink/sdl_core/commit/5da57703e02fdf526c300cc26a45ff75f7d1ffd0)

### Summary
- Change `.alert-top h4` overflow value to "auto". The previous value ("scroll") would always display scrollbars, whether or not any content is actually clipped.
- Fix deafult click handler for grayed out app in app store

##### Bug Fixes
Currently clicking an installed(grayed out) app tile in the app store results in a console error
```
Uncaught TypeError: p is not a function
    at onClick (HScrollMenuItem.js:73)
    at onClick (Link.js:37)
    at Object.u (react-dom.production.min.js:14)
    at d (react-dom.production.min.js:14)
    at react-dom.production.min.js:14
    at y (react-dom.production.min.js:15)
    at at (react-dom.production.min.js:52)
    at ot (react-dom.production.min.js:51)
    at ut (react-dom.production.min.js:52)
    at dt (react-dom.production.min.js:56)
```
This PR fixes the default click handler for the `menuItem.greyOut` case

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

iCollin commented on src/css/components/_alert.scss:
```diff
@@ -63,7 +63,7 @@
 
     h4 {
-        overflow: scroll;
+        overflow: auto;
```
With auto I cannot scroll down on an alert with too much text.

For ex when I send this alert I can tell the text continues down past the bottom of the container but I cannot read that text or scroll down to it.

```
 Alert {"alertText1":"Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo. Nullam dictum felis eu pede mollis pretium. Integer tincidunt. Cras dapibu"}
```

ShobhitAd replied:
I'm not seeing a scroll bar on my end, even on the latest relase/0.11.0 commit, for an alert request with that value.

I don't think that alertText* should be affected by this PR since the change relates to `h4` whereas all alert text fields are `p` elements
![Screenshot from 2021-09-14 22-50-55](https://user-images.githubusercontent.com/10703941/133362757-2ddbef83-1f8a-41ab-a766-9afe2edf56ba.png)

iCollin commented:
It seems to me that this bug is due to my change as part of the Scrollable Message implementation: https://github.com/smartdevicelink/generic_hmi/pull/394/files#diff-086204f81abae904e396dcfbfcfe45acdffa9541673ff56447fe45b3b8063118R10

I made it possible for the Scrollable Message scroll bar to still be visible, not realizing that it will affect other interactions. I suppose this should be modified to affect a smaller scope.

ShobhitAd commented:
> It seems to me that this bug is due to my change as part of the Scrollable Message implementation: https://github.com/smartdevicelink/generic_hmi/pull/394/files#diff-086204f81abae904e396dcfbfcfe45acdffa9541673ff56447fe45b3b8063118R10
> 
> I made it possible for the Scrollable Message scroll bar to still be visible, not realizing that it will affect other interactions. I suppose this should be modified to affect a smaller scope.

Unfortunately, I'm not able to overwrite the `webkit-scrollbar` display property in the `.scrollableText` class after reverting the change https://github.com/smartdevicelink/generic_hmi/pull/394/files#diff-086204f81abae904e396dcfbfcfe45acdffa9541673ff56447fe45b3b8063118R10.

I did find that removing https://github.com/smartdevicelink/generic_hmi/pull/441/files#diff-086204f81abae904e396dcfbfcfe45acdffa9541673ff56447fe45b3b8063118L10 did not affect the visibility of scrollbars for other interactions (Alert, AppStore, ScrollableMessage, etc.) so I removed that section and changed the overflow value for `.appstore-menu` to "hidden" in https://github.com/smartdevicelink/generic_hmi/pull/441/commits/befb015cf5945ceb1f9d6a25e21c645f2af0cfd8

iCollin commented on src/css/components/_appstore-menu.scss:
```diff
@@ -1,15 +1,12 @@
     width: 100%;
     height: $master-height - 75px;
-    overflow-y: scroll;
+    overflow-y: hidden;
```
```suggestion
    overflow-y: scroll;
```

It should still be possible to scroll here, right?

iCollin reviewed (commented):
looks good otherwise!
